### PR TITLE
[TACHYON-972]  Remove unused imports from LocalBlockOutStream.

### PR DESCRIPTION
**Pull request by LeoOnHack**
#TACHYON-972 Remove unused imports from LocalBlockOutStream.

https://tachyon.atlassian.net/browse/TACHYON-972

### CODE REVIEW

**Comment by LeoOnHack** [comment]
I am very interested in Tachyon and would love to continue contribute to this community. 
Following the instructions on new contributor tasks I edited Two of new contributor tasks real quick. 

But I got a little issue here, as you can see both minor changes got included in One pull request.

Remove unused imports from LocalBlockOutStream.  Tachyon-972
Add log for file block creation in LocalBlockOutStream.  Tachyon-745

Do I have to undo the second change and wait for the 1st pull request get merged and redo the second change?  

Surely for later bigger changes I will submit them one by one. :)
Thanks!

~Leo(Biao) Ma

File: clients/unshaded/src/main/java/tachyon/client/block/LocalBlockOutStream.java [hunk 1]
```diff
@@ -37,6 +38,7 @@
  * thread and are not thread safe.
  */
 public class LocalBlockOutStream extends BufferedBlockOutStream {
+  private static final Logger LOG = LoggerFactory.getLogger(Constants.LOGGER_TYPE);
```
Comment (yupeng9) [inline clients/unshaded/src/main/java/tachyon/client/block/LocalBlockOutStream.java]: As a convention, we have a blank line between static variables and member variables.

**Comment by yupeng9** [comment]
Jenkins, test this please.

**Comment by yupeng9** [comment]
@LeoOnHack , welcome to the community! For minor changes, it's fine to combine them into one commits. But as you said, for larger ones, please open one PR at a time in the future. Also, please update the PR title to include both ticket numbers.

Hope to see more contributions from you :)

**Comment by gpang** [comment]
ok to test

**Comment by gpang** [comment]
Thanks for the patch @LeoOnHack !

**Comment by gpang** [comment]
LGTM

**Pull request merged**